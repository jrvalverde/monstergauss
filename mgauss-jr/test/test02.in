*MOL RHF=NORESET, BA=G, GE=B, CI
*PRT IGC=Y
 NH2 2A1
 0 2
   7
   1 1.78083                                -.577080000
   1 -1.78083000                            -.577080000

6111410310310E
 1
     1S 1.0
5909.44             0.002004
887.451             0.015310
204.749             0.074293
59.8376             0.253364
19.9981             0.600576
2.6860              0.245111
     2S 1.0
7.1927              1.0
     3S 1.0
0.70000             1.0
     4S 1.0
0.2133              1.0
     1P 1.0
26.7860             0.0                 0.018257
5.9564              0.0                 0.116407
1.7074              0.0                 0.390111
0.5314              0.0                 0.637221
     2P 1.0
0.1654              0.0                 1.0
****
 2 3
     1S  1.0
19.2406             .032828
2.8992              .231208
.6534               .817238
     2S  1.0
.1776               1.0
****

FFFAF
11111


 1 1 4 3 1 5 8 5 7 8 5 5 8 5
   4
 A1    8   3   1
 A2    0   0   0
 B1    2   1   0
 B2    4   1   0
 0 2 1 2 2
