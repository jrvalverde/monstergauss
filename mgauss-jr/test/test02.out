 WELCOME TO U OF T MONSTERGAUSS                                                                         UNX VERSION: 13 AUG 1991
 ******************************
0CONTROL CARDS:

 *MOL RHF=NORESET, BA=G, GE=B, CI                                        
 *PRT IGC=Y                                                              

0SEQUENCE    GROUP     MG OPTIONS         GROUP OPTIONS (USER CONTROLLABLE)                  LINKS TO BE CALLED          JUMP

     1         2      000037012100 000000000000000000000000000000000000000000000000   01000000000000000000000000000000     0
     2         3      000037012100 004000010020000000000000000000000000000000000000   11000100000000000000000000000000     0
     3         4      000037012100 100000010000000010000000000000000000000000000000   11000000000000000000000000000000     0
     4         5      000037012100 100000000000000020000000000000000000000000000000   00100000000000000000000000000000     0
     5         6      000037012100 070000000000000000001000000000000000000000000000   10000000000000000000000000000000     0
     7         9      000037012100 000000000000000000000000000000000000000000000000   11111100000000000000000000000000     0
0OPTION NUMBERS:      000000000111 111111122222222223333333333444444444455555555556   66666666677777777778888888888999
                      123456789012 345678901234567890123456789012345678901234567890   12345678901234567890123456789012

0CONVERSION FACTORS:  1 BOHR =  .52917706 ANGSTROMS
0                     1 HARTREE/BOHR = 8.2388584 MDYNE

                      1 HARTREE/ANGSTROM**2 = 4.3598149 MDYNE/ANGSTROM

                      1 ELECTRON*BOHR = 2.5417655 DEBYE

                      1 HARTREE/PARTICLE = 627.50959 KCAL/MOL = 2625.5001 KJ/MOL = 27.211611 EV/PARTICLE = 219474.67 CM(-1)/PARTICLE

                      1 ATOMIC UNIT OF ELECTRIC FIELD = 5.1422506D+09 VOLTS/CM
1 NH2 2A1                                                                
 ------------------------------------------------------------------------
0CHARGE  =   0          MULTIPLICITY  =  2



 ------------------------------------------------------------------------------------------------------------------------------------
                    COORDINATES IN ANGSTROMS                               COORDINATES IN BOHR                    ATOMIC ISOTOPE/MASS
   I   AN        X             Y             Z                        X             Y             Z               MASS#    MASS (AMU)
 ------------------------------------------------------------------------------------------------------------------------------------
   1    7     .0000000      .0000000      .0000000                 .0000000      .0000000      .0000000             14    14.00307401
   2    1     .9423744      .0000000     -.3053775                1.7808300      .0000000     -.5770800              1     1.00782504
   3    1    -.9423744      .0000000     -.3053775               -1.7808300      .0000000     -.5770800              1     1.00782504
 ------------------------------------------------------------------------------------------------------------------------------------

0PROJECTING FROM STO-3G BASIS (5D/7F)

0BASIS SET TYPE: EXTERNAL GENERAL     ( 5D/ 7F)
 POLARIZATION  : UNKNOWN                                                                              
 DIFFUSE FUNCS : UNKNOWN                                                                              
1BASIS SET FOR THIS CALCULATION (EXPONENTS ARE SCALED)

 **********************************************************************************************************************************
               ATOMIC ORBITAL                                     GAUSSIAN FUNCTIONS
 **********************************************************************************************************************************
      ATOM       NO.    TYPE   SCALE FACTOR    EXPONENT        S COEF         P COEF         D COEF         F COEF
 **********************************************************************************************************************************
     NITROGEN
                    1    1S       1.00000
                                             5.9094400D+03  2.0040000D-03   .0000000D+00   .0000000D+00   .0000000D+00
                                             8.8745100D+02  1.5310000D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             2.0474900D+02  7.4293000D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             5.9837600D+01  2.5336400D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             1.9998100D+01  6.0057600D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             2.6860000D+00  2.4511100D-01   .0000000D+00   .0000000D+00   .0000000D+00
                    2    2S       1.00000
                                             7.1927000D+00  1.0000000D+00   .0000000D+00   .0000000D+00   .0000000D+00
                    3    3S       1.00000
                                             7.0000000D-01  1.0000000D+00   .0000000D+00   .0000000D+00   .0000000D+00
                    4    4S       1.00000
                                             2.1330000D-01  1.0000000D+00   .0000000D+00   .0000000D+00   .0000000D+00
                5-  7    1P       1.00000
                                             2.6786000D+01   .0000000D+00  1.8257000D-02   .0000000D+00   .0000000D+00
                                             5.9564000D+00   .0000000D+00  1.1640700D-01   .0000000D+00   .0000000D+00
                                             1.7074000D+00   .0000000D+00  3.9011100D-01   .0000000D+00   .0000000D+00
                                             5.3140000D-01   .0000000D+00  6.3722100D-01   .0000000D+00   .0000000D+00
                8- 10    2P       1.00000
                                             1.6540000D-01   .0000000D+00  1.0000000D+00   .0000000D+00   .0000000D+00
 ----------------------------------------------------------------------------------------------------------------------------------
     HYDROGEN
                   11    1S       1.00000
                                             1.9240600D+01  3.2828000D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             2.8992000D+00  2.3120800D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             6.5340000D-01  8.1723800D-01   .0000000D+00   .0000000D+00   .0000000D+00
                   12    2S       1.00000
                                             1.7760000D-01  1.0000000D+00   .0000000D+00   .0000000D+00   .0000000D+00
 ----------------------------------------------------------------------------------------------------------------------------------
     HYDROGEN
                   13    1S       1.00000
                                             1.9240600D+01  3.2828000D-02   .0000000D+00   .0000000D+00   .0000000D+00
                                             2.8992000D+00  2.3120800D-01   .0000000D+00   .0000000D+00   .0000000D+00
                                             6.5340000D-01  8.1723800D-01   .0000000D+00   .0000000D+00   .0000000D+00
                   14    2S       1.00000
                                             1.7760000D-01  1.0000000D+00   .0000000D+00   .0000000D+00   .0000000D+00
 **********************************************************************************************************************************

0*** THE S=P EXPONENT CONSTRAINT HAS BEEN REMOVED
0RYSSET ROOT STATISTICS:     INTERPOLATED:      434     CALCULATED:        0     ASSYMPTOTIC APPROXIMATION:      406
0STANDARD CUTOFFS SELECTED IN SHELL.
0SHELL: FMTGEN WAS CALLED     1732 TIMES


       3009 TWO ELECTRON INTEGRALS WRITTEN OUT ON UNIT  9 WITH INTEGRAL CUTOFF  1.00000D-10
1RHF OPEN SHELL SPECIFICATION
0GROUP     : FFFAF         
 DEGENERACY: 11111000000000
 ALPHA OCC : 00000000000000
 BETA OCC  : 00000000000000

0ORBITAL GROUP :       F       D       S       A       V
0NUMBER OF MO  :       4       0       0       1       9

0LEVEL    DEGENERACY    DOUBLE OCC    SINGLE ALPHA    SINGLE BETA    CONFIGURATIONS    SAMPLE CONFIGURATION
0  1           1             0              1               0               1          A   
+                                                                                      __  

0COULOMB COUPLING COEFFICIENTS   A(I,J)
0  MO          OCCA      OCCB         CLOSED   OPEN 1   
0CLOSED       4.0000    4.0000        2.0000   1.0000
 OPEN 1       1.0000     .0000        1.0000    .0000

0EXCHANGE COUPLING COEFFICIENTS   B(I,J)
0  MO          OCCA      OCCB         CLOSED   OPEN 1   
0CLOSED       4.0000    4.0000        1.0000    .5000
 OPEN 1       1.0000     .0000         .5000    .0000

0PH INITIAL GUESS ... SCALE FACTOR (K/2) =  .8750
 SMALLEST EIGENVALUE OF OVERLAP IS  3.05361D-01
 SMALLEST EIGENVALUE OF OVERLAP IS  3.14053D-02
 COEFFICIENTS PROJECTED TO CURRENT BASIS
1INITIAL GUESS COEFFICIENTS         5 ALPHA SPIN ELECTRONS,    4 BETA SPIN ELECTRONS

                           1          2          3          4          5          6          7          8          9         10
 
    EIGENVALUES---   -15.68101   -1.24783    -.60057    -.42506    -.42000     .28485    1.39900     .00000     .00000     .00000
 
   1  1   N   1S       -.59794    -.10481     .00000    -.00761     .00000     .00000     .08078     .00000    1.74322    -.18551
   2  1   N   2S       -.45294    -.15645     .00000    -.01678     .00000     .00000     .18276     .00000   -1.85234     .51950
   3  1   N   3S        .02943     .42475     .00000     .05583     .00000     .00000    -.65900     .00000     .15828   -1.74834
   4  1   N   4S       -.01479     .43933     .00000     .04233     .00000     .00000    -.69969     .00000     .16144    3.35997
   5  1   N   1PX       .00000     .00000    -.44229     .00000     .00000    1.06134     .00000     .60181     .00000     .00000
   6  1   N   1PY       .00000     .00000     .00000     .00000     .89215     .00000     .00000     .00000     .00000     .00000
   7  1   N   1PZ       .00099     .02978     .00000     .87852     .00000     .00000     .26707     .00000     .05283     .13135
   8  1   N   2PX       .00000     .00000    -.12456     .00000     .00000     .30663     .00000   -3.55519     .00000     .00000
   9  1   N   2PY       .00000     .00000     .00000     .00000     .18175     .00000     .00000     .00000     .00000     .00000
  10  1   N   2PZ       .00151    -.00493     .00000     .17960     .00000     .00000     .07537     .00000    -.10907    -.58270
 
  11  2   H   1S        .00203     .14982    -.29384    -.03325     .00000    -.51684     .53211    -.28330     .08323     .23637
  12  2   H   2S        .00668     .07202    -.17615    -.01532     .00000    -.52147     .49016    3.02421    -.20222   -1.50289
 
  13  3   H   1S        .00203     .14982     .29384    -.03325     .00000     .51684     .53211     .28330     .08323     .23637
  14  3   H   2S        .00668     .07202     .17615    -.01532     .00000     .52147     .49016   -3.02421    -.20222   -1.50289
 

                          11         12         13         14
 
    EIGENVALUES---      .00000     .00000     .00000     .00000
 
   1  1   N   1S        .22914     .00000     .02922     .00000
   2  1   N   2S       -.06783     .00000    -.00557     .00000
   3  1   N   3S       -.77120     .00000    -.05647     .00000
   4  1   N   4S        .41223     .00000     .00617     .00000
   5  1   N   1PX       .00000     .00000     .00000     .10407
   6  1   N   1PY       .00000    -.76777     .00000     .00000
   7  1   N   1PZ      -.28490     .00000    -.67906     .00000
   8  1   N   2PX       .00000     .00000     .00000    -.85595
   9  1   N   2PY       .00000    1.16291     .00000     .00000
  10  1   N   2PZ       .34642     .00000    1.11772     .00000
 
  11  2   H   1S       -.70097     .00000     .32416     .84461
  12  2   H   2S        .68654     .00000    -.16394    -.44579
 
  13  3   H   1S       -.70097     .00000     .32416    -.84461
  14  3   H   2S        .68654     .00000    -.16394     .44579
 
 
 THESE MO COEFFICIENTS HAVE BEEN WRITTEN TO FILE 25, RE-ORDERED BY GROUPS
 THESE MO COEFFICIENTS HAVE BEEN WRITTEN TO FILE 27, RE-ORDERED BY GROUPS
1RHF OPEN SHELL SCF               NUCLEAR REPULSION ENERGY IS      7.759408102  HARTREES
0CONVERGENCE ON DENSITY MATRIX REQUIRED TO EXIT IS  1.0000D-06,  INITIAL SCALE FACTOR (SCALST) IS  .80000
 SCALE FACTOR CUT RATIO (FACTDN) IS  .66667,  SCALE FACTOR INCREASE RATIO (FACTUP) IS 1.20000
 SCALE FACTOR LOWER LIMIT (SLIMDN) IS  .00050,  SCALE FACTOR UPPER LIMIT (SLIMUP) IS  .80000
 SWITCH TO INCREASE MODE AT D CONVERGENCE  2.0000D-03 (SCRIT, RHF=INCR MODE ONLY).
0 CYCLE                   ENERGY                    SCF CONVERGENCE         CI CONVERGENCE         EXTRAPOLATION     SCALE      RESET
0 CI SCF       ELECTRONIC           TOTAL          COEFF      DENSITY      COEFF      DENSITY

   1   0      -63.115235901      -55.355827800             
   1   1      -62.702824747      -54.943416646              4.3193D-02                                             5.3333D-01    NO  
   1   2      -63.306985879      -55.547577778   1.4808D-02 1.4209D-02                                             5.3333D-01    NO  
   1   3      -63.233936061      -55.474527960   8.2648D-03 8.6591D-03                                             5.3333D-01    NO  
   1   4      -63.278796665      -55.519388564   4.3194D-03 4.7602D-03                                             5.3333D-01    NO  
   1   5      -63.260539689      -55.501131588   3.2434D-03 3.4486D-03                                             5.3333D-01    NO  
   1   6      -63.270451456      -55.511043354   1.6891D-03 1.9896D-03                                             5.3333D-01    NO  
   1   7      -63.264098429      -55.504690327   1.4124D-03 1.4814D-03                                             5.3333D-01    NO  
   1   8      -63.267066763      -55.507658661   7.4630D-04 8.7869D-04                                             5.3333D-01    NO  
   1   9      -63.264874380      -55.505466279   6.5715D-04 6.6686D-04                                             5.3333D-01    NO  
   1  10      -63.265787977      -55.506379876   3.7072D-04 4.1094D-04                                             5.3333D-01    NO  
   1  11      -63.265032754      -55.505624653   3.2948D-04 3.1728D-04                                             5.3333D-01    NO  
   1  12      -63.265307950      -55.505899849   2.0493D-04 2.0587D-04                                             5.3333D-01    NO  
   1  13      -63.265045117      -55.505637015   1.7834D-04 1.6111D-04                                             5.3333D-01    NO  
   1  14      -63.265124269      -55.505716168   1.2167D-04 1.1066D-04                                             5.3333D-01    NO  
   1  15      -63.265031069      -55.505622968   1.0282D-04 8.7277D-05                                             5.3333D-01    NO  
   1  16      -63.265051812      -55.505643711   7.4957D-05 6.3016D-05                                             5.3333D-01    NO  
   1  17      -63.265017842      -55.505609740   6.1849D-05 4.9859D-05                                3-POINT      5.3333D-01    NO  
   1  18      -63.265043451      -55.505635350              2.6807D-05                                             5.3333D-01    NO  
   1  19      -63.264946600      -55.505538498   1.3303D-05 1.5138D-05                                             5.3333D-01    NO  
   1  20      -63.265010253      -55.505602151   1.3057D-05 1.2279D-05                                             5.3333D-01    NO  
   1  21      -63.264979415      -55.505571313   6.5290D-06 7.3651D-06                                             5.3333D-01    NO  
   1  22      -63.265000763      -55.505592662   6.5461D-06 6.0736D-06                                4-POINT      5.3333D-01    NO  
   1  23      -63.265001983      -55.505593881              1.6376D-06                                             5.3333D-01    NO  
   1  24      -63.264996687      -55.505588586   9.5607D-07 8.7775D-07                                             5.3333D-01    NO  
   1  25      -63.265001600      -55.505593498   1.5416D-06 1.1853D-06                                             8.0000D-01    YES 
   1  26      -63.264993346      -55.505585245   6.9323D-07 5.2844D-07                                             8.0000D-01    YES 
   1  27      -63.264998231      -55.505590130                                       .0000D+00                     8.0000D-01    YES 
0AT TERMINATION TOTAL ENERGY IS      -55.505590  HARTREES          ORBITAL CONVERGENCE IS 4.53228D-07

0S =  .500 (  .500 EXPECTED),   S(S+1) =   .7500 (   .7500 EXPECTED)
1MOLECULAR ORBITALS         5 ALPHA SPIN ELECTRONS,   4 BETA SPIN ELECTRONS

                           1          2          3          4          5          6          7          8          9         10
 
    EIGENVALUES---   -15.54894   -1.08050    -.70474    -.39707    -.26207     .24372     .36228     .69282     .70200     .74055
 
   1  1   N   1S        .59349    -.12711     .00000     .00000    -.03751    -.06368     .00000     .00319     .00000     .00000
   2  1   N   2S        .44588    -.17228     .00000     .00000    -.05161    -.08548     .00000     .00191     .00000     .00000
   3  1   N   3S        .00240     .48635     .00000     .00000     .16027     .16628     .00000     .05532     .00000     .00000
   4  1   N   4S        .00085     .41296     .00000     .00000     .21810    1.67218     .00000    -.25228     .00000     .00000
   5  1   N   1PX       .00000     .00000    -.54931     .00000     .00000     .00000     .43543     .00000     .00000    -.67836
   6  1   N   1PY       .00000     .00000     .00000     .70325     .00000     .00000     .00000     .00000     .94385     .00000
   7  1   N   1PZ      -.00056    -.10138     .00000     .00000     .70276    -.14668     .00000    -.86312     .00000     .00000
   8  1   N   2PX       .00000     .00000    -.12259     .00000     .00000     .00000    2.00201     .00000     .00000    2.93312
   9  1   N   2PY       .00000     .00000     .00000     .43097     .00000     .00000     .00000     .00000   -1.09529     .00000
  10  1   N   2PZ      -.00005    -.03191     .00000     .00000     .34167    -.43977     .00000    1.15075     .00000     .00000
 
  11  2   H   1S        .00031     .17007    -.25027     .00000    -.08259    -.09089     .04246     .17742     .00000    -.29839
  12  2   H   2S       -.00017     .02749    -.13639     .00000    -.05939   -1.25217   -2.30654     .02762     .00000   -1.59506
 
  13  3   H   1S        .00031     .17007     .25027     .00000    -.08259    -.09089    -.04246     .17742     .00000     .29839
  14  3   H   2S       -.00017     .02749     .13639     .00000    -.05939   -1.25217    2.30654     .02762     .00000    1.59506
 

                          11         12         13         14
 
    EIGENVALUES---      .95603    1.28995    1.41257   33.01572
 
   1  1   N   1S       -.04002     .14747     .00000   -1.76195
   2  1   N   2S       -.07757     .24697     .00000    1.91403
   3  1   N   3S        .76727   -1.80766     .00000    -.38607
   4  1   N   4S      -1.15838    2.76107     .00000     .35608
   5  1   N   1PX       .00000     .00000    -.86230     .00000
   6  1   N   1PY       .00000     .00000     .00000     .00000
   7  1   N   1PZ       .32448     .17430     .00000     .00665
   8  1   N   2PX       .00000     .00000     .92483     .00000
   9  1   N   2PY       .00000     .00000     .00000     .00000
  10  1   N   2PZ      -.05037    -.34584     .00000    -.03431
 
  11  2   H   1S        .81828     .46918     .99681     .02727
  12  2   H   2S       -.25099   -1.18134   -1.32839    -.10937
 
  13  3   H   1S        .81828     .46918    -.99681     .02727
  14  3   H   2S       -.25099   -1.18134    1.32839    -.10937
 
 
 THE FIRST   4 MO ARE DOUBLY OCCUPIED FROZEN CORE ORBITALS
0OPEN SHELL ORBITAL OCCUPANCIES:    MO     OCCA     OCCB
                                     5    1.0000    .0000
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (ALPHA SPIN)     (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1          2          3
 
         1   N         4.02801     .15773     .15773
 
         2   H          .15773     .18098    -.01045
 
         3   H          .15773    -.01045     .18098
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (BETA SPIN)      (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1          2          3
 
         1   N         3.04721     .16261     .16261
 
         2   H          .16261     .16393    -.01277
 
         3   H          .16261    -.01277     .16393
 
1MULLIKEN POPULATION MATRIX CONDENSED TO ATOMS (TOTAL)          (DOUBLE THE OFF-DIAGONAL ELEMENTS FOR 'BOND ORDERS')

                           1          2          3
 
         1   N         7.07522     .32035     .32035
 
         2   H          .32035     .34491    -.02322
 
         3   H          .32035    -.02322     .34491
 
0ATOMIC CHARGES (ALPHA(1), BETA(2)), ATOMIC SPIN DENSITIES(3), ATOMIC CHARGES (TOTAL(4), NET(5))

                           1          2          3          4          5
 
         1   N         4.34347    3.37244     .97103    7.71591    -.71591
 
         2   H          .32826     .31378     .01449     .64204     .35796
 
         3   H          .32826     .31378     .01449     .64204     .35796
 
1BOND ORDER AND VALENCE INDICES (I. MAYER, I.J.Q.C., 29, 477-483 (1986))


          BOND ORDER MATRIX

                           1          2          3
 
         1   N          .00000     .82817     .82817
 
         2   H          .82817     .00000     .01814
 
         3   H          .82817     .01814     .00000
 
0                      VALENCE  FREE VALENCE

                           1          2
 
         1   N         2.59924     .94290
 
         2   H          .84652     .00021
 
         3   H          .84652     .00021
 
1*********************************
 *INTEGRAL TRANSFORMATION PACKAGE*
 *********************************

0GROUP SPECIFICATION:

   1  1  4  3  1  5  8  5  7  8  5  5  8  5

0TRANSFORM 14 MO
1****************************
 *CI CONFIGURATION GENERATOR*
 ****************************


 OPEN SHELL DOUBLET


 SYMMETRY     NUMBER OF MO OF THAT SYMMETRY
  OF MO       TOTAL     OCCUPIED     FROZEN VIRTUAL
 --------------------------------------------------
   A1           8           3              1
   A2           0           0              0
   B1           2           1              0
   B2           4           1              0


 OCCUPANCY VECTOR:
   0  2  1  2  2
0NUMBER OF ACTIVE OCCUPIED MO                        4
 POSITION OF FIRST SINGLY OCCUPIED OR VIRTUAL MO     4
 TOTAL NUMBER OF ACTIVE MO                          12
 TOTAL NUMBER OF DOUBLY OCCUPIED ACTIVE MO           3
 NUMBER OF SYMMETRY TYPES                            4
0TOTAL NUMBER OF CONFIGURATIONS                    189

0NUMBER OF 3360 WORD BLOCKS WRITTEN TO THE FORMULA TAPE:        15
1********************
 *CI DIAGONALIZATION*
 ********************



 ITERATION     CORRELATION ENERGY

      2             -.0896144
      3             -.0927448
      4             -.0929123
      5             -.0929207
      6             -.0929211

0TOTAL SCF ENERGY                          -55.5055901
 CORRELATION ENERGY                          -.0929211
0TOTAL CI ENERGY                           -55.5985112
0TOTAL CI ENERGY + (1-C0*C0)*ECORR         -55.6021561
0WHERE (1-C0*C0)*ECORR IS AN APPROXIMATE CORRECTION TO THE CI ENERGY FOR QUADRUPLE EXCITATIONS
0C0 IS THE COEFFICIENT OF THE ROOT CONFIGURATION
 C0 =     .9801907

0NORMAL TERMINATION, TOTAL CPU TIME FOR THIS CALCULATION WAS      .070 MINUTES.
3.7u 0.6s 0:06 72%
